articles/how-to-syn-and-update-a-forked-repo/index.md

```diff
@@ -0,0 +1,571 @@
+status: publish
+published: true
+url: /engineering-education/how-to-syn-and-update-a-forked-repo/
+title: How to Sync and Update a Forked Repository
+description: This article shows you how to sync and update a forked repo using Github desktop and git bash with examples.
+author: joseph-chege
+date: 2020-09-11T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:		
+  - url: /engineering-education/how-to-syn-and-update-a-forked-repo/hero.jpg
+    alt: How to sync to forked repo
+---
+This article assumes that you have a GitHub account. [Check instructions](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account) on how to create a GitHub user account. In this article, we shall briefly look at how we can submit work and contributions to a GitHub team project. Before we get started, you need to understand the following terms, as you will constantly come across them in daily GitHub workflow.
+<!--more-->
+### Getting started
+-   **Fork** is a copy of a repository of someone's else project. Forks allows you have to have a repository of a project that you do not own on your GitHub account so that you can make some changes without affecting the original repository. Once the changes on your forked repository are validated, you can submit these changes as contributions to the original project. Forks allows you to make your contributions to a team project.
+-   **Remote** a repository on GitHub account exists as a remote repository.
+-   **Clone** is downloading a copy of the remote repository to local computer.
+-   **Commits** changes made to a repository. Committing saves edits/changes to our GitHub repository. With GitHub, every commit you make has a specific signature/ID the keeps the record of the changes you have made. If you happen to make a wrong change, you can revert the commit and your remote repository will be as it was before you made the changes. A commit has a commit message that allows you to have a brief description of the changes you are making.
+-   **Push** is used to transfer commits made on local repository to GitHub remote repository. To push to GitHub you make a push request to update your local commits to your remote repository.
+-   **Branch** a feature branch or base branch. A feature branch typically helps you to isolate your changes from the other team as you make and test this changes. The base branch consists of merged feature branches.
+-   **Pull** is to notify the project team of the changes you have made from a pushed branch in your GitHub repository or basically requesting the owner of the repository to pull changes you made, thus called a pull request. A collaborator can be assigned to review your pull and propose any potential changes you need to make. Review changes are done with commits. Once changes and reviews are complete a team contributor or you (with access), can merge your branch to the base branch of the original repository.
```

Comment:
Pulls are not Pull Requests. Change this to pull request and you'd be right. Pull is requesting the diff from a branch